src/ontology/mondo-edit.obo

```diff
@@ -343439,19 +343443,28 @@ synonym: "mental retardation, autosomal dominant 42" EXACT DEPRECATED [OMIM:6169
 synonym: "mental retardation, autosomal dominant type 42" EXACT DEPRECATED [MONDORULE:2, OMIM:616973]
 synonym: "MRD42" EXACT ABBREVIATION [DOID:0070072, OMIM:616973]
 xref: DOID:0070072 {source="MONDO:equivalentTo"}
+xref: GARD:17893 {source="MONDO:GARD"}
 xref: GARD:18501 {source="MONDO:GARD"}
```

Comment:
there are 2 GARD x-ref. Can one of these be "preferred external"?

Reply:
@sabrinatoro GARD:18501 is the preferred external

Reply:
Never mind. These are not equivalent.